cmd/jaeger/integration/receivers/storagereceiver/config.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package storagereceiver
+
+import (
+	"github.com/asaskevich/govalidator"
+
+	grpcCfg "github.com/jaegertracing/jaeger/plugin/storage/grpc/config"
+)
+
+type Config struct {
+	GRPC grpcCfg.Configuration `mapstructure:"grpc"`
```

Comment:
similar to storage exporter, this should be just a name of the storage that can be retrieved from `jaeger_storage_extension`